### PR TITLE
Add SellingPoints component

### PR DESCRIPTION
**Pull request by rosslh**
## Proposed Change

Adds a component for the `Grow Your Network. Showcase to Experts.` section!

### How did you do this?

- Create SellingPoints component
- Create SellingPointsIcons sub-component
- Make it responsive

### Why are you choosing this approach?

I chose to create a separate component for the icons so that our code doesn't repeat itself and adding icons would be as simple as passing in an image, string, and number.

### Any open questions you want to ask code reviewers?

- Since we don't have mocks for mobile I just coded what I think looks good. Lmk if you'd change anything

### List of changes:

  - Create SellingPoints component
  - Create Icons component in same file (only used locally)
  - Add icons to img folder

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new tests for my core changes (where applicable).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit 555f6663f53fe72a5e1140372c77e0d4ecc2c4d2

https://deploy-preview-100--qhacks-website.netlify.com

File: src/components/SellingPoints.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import mentors from "../assets/img/mentors.svg";
+import hackers from "../assets/img/hackers.svg";
+import judges from "../assets/img/judges.svg";
+import schools from "../assets/img/schools.svg";
+import applications from "../assets/img/applications.svg";
+import challenges from "../assets/img/challenges.svg";
+import { css, nthChild } from "glamor";
```
Comment (RobertWSaunders) [inline src/components/SellingPoints.js]: Add a newline after the imports, nitpick!

File: src/components/SellingPoints.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  maxWidth: "500px"
+};
+
+const imgStyle = {
+  backgroundColor: "#515151",
+  height: "320px",
+  maxWidth: "475px"
+};
+
+const Icons = (props) => (
```
Comment (RobertWSaunders) [inline src/components/SellingPoints.js]: Let's break this down into its own file, not a fan of having a `Icons` component live within the `SellingPoints` file.

File: src/components/SellingPoints.js [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+const flexChildStyle = css(
+  {
+    flex: 1,
+    textAlign: "center",
+    minWidth: "300px"
+  },
+  nthChild("2n", {
+    margin: "0 28px 32px 28px"
+  }),
+  nthChild("2n-1", {
```
Comment (RobertWSaunders) [inline src/components/SellingPoints.js]: I'm curious what this is doing?

Comment (rosslh) [reply src/components/SellingPoints.js]: Nice catch, that logic is no longer needed

**Comment by RobertWSaunders** [comment]
Also, can we fully fill out the PR templated please!

**Comment by rosslh** [comment]
I don't know why it's saying coverage is decreasing for ActionButton when I didn't make any changes to that component in this branch

File: src/components/SellingPoints.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      css={{
+        ...twoColumnStyle,
+        flexWrap: "wrap"
+      }}
+    >
+      <div css={{ ...flexChildStyle, ...detailsStyle }}>
+        <h1>Grow Your Network.</h1>
+        <p css={blurbStyle}>
+          QHacks was first held in 2016 with a mission to advocate and incubate
+          the tech community at Queen's University and throughout Canada. Going
```
Comment (RobertWSaunders) [inline src/components/SellingPoints.js]: Can we delimit these so the diffs don't look terrible?

Comment (RobertWSaunders) [reply src/components/SellingPoints.js]: As an update to this, we can put `&#39;` in place!

**Comment by RobertWSaunders** [comment]
Another question I have with this, do we have pictures to go into those placeholders?

**Pull request closed without merging**